t/APISIX.pm

```diff
@@ -502,6 +503,8 @@ _EOC_
 
     require "resty.core"
 
+    $extra_init_by_lua_start
```

Comment:
Could we just move $extra_init_by_lua to this place?

Reply:
Ping @monkeyDluffy6017

Reply:
Done